sympy/external/tests/test_numba.py

```diff
@@ -0,0 +1,29 @@
+# This testfile tests SymPy's compatibility with Numba
+
+from sympy.external import import_module
+from sympy import sin, exp, symbols, lambdify, Matrix
+
+numba = import_module('numba')
+if not numba:
+    #bin/test will not execute any tests now
+    disabled = True
+
+def test_jit():
+    # Test that numba.jit works with lambdify
+    x = symbols('x')
+    f1 = lambdify(x, sin(x), 'numpy')
+    assert abs(numba.jit(f1)(1) - f1(1)) < 1e-15
+
+    y = exp(-x)
+    tanh = (1.0 - y) / (1.0 + y)
+    ddx_tanh = tanh.diff(x)
+    f2 = lambdify(x, ddx_tanh, 'numpy')
+    assert abs(numba.jit(f2)(1) - f2(1)) < 1e-15
+
+    from itertools import chain
+    X = Matrix(symbols('x:3')).T
+    Y = Matrix(symbols('y:3')).T
+    f3 = lambdify((*X, *Y), X+Y)
+    xs = (1,2,3,4,5,6)
+    element_comp = abs(f3(*xs) - numba.njit(f3)(*xs) < 1e-15)
```

Comment:
This is calling `abs` on the bool array. The `1e-15` shouldn't be inside the bracket.